src/components/MessageInput/MessageInput.tsx

```diff
@@ -473,49 +482,20 @@ export const MessageInputWithContext = <
             <Input
               additionalTextInputProps={additionalTextInputContainerProps}
               getUsers={getUsers}
-              handleOnPress={handleOnPress}
+              handleOnPress={toggleAttachmentPicker}
             />
           ) : (
             <>
```

Comment:
this outter `<></>` probably isn't necessary anymore

Reply:
It is, there are multiple view inside